### ISSUE TITLE
Warning: DatePickerIOS has been merged with DatePickerAndroid and will be removed in a future release.

### ISSUE THREAD

**Issue by cspicuzza** [henninghall/react-native-date-picker#112]
**Describe the bug**
The following warning shows in the console: 
```Warning: DatePickerIOS has been merged with DatePickerAndroid and will be removed in a future release. It can now be installed and imported from '@react-native-community/datetimepicker' instead of 'react-native'```

**To Reproduce**

```import React, {PureComponent} from 'react';
import {View, Text, TouchableOpacity} from 'react-native';
import {styles} from './styles';
import DatePicker from 'react-native-date-picker';

class DatePickerTest extends PureComponent {
  state = {
    date: new Date(),
  };
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.title}>react-native-date-picker</Text>
        <DatePicker
          date={this.state.date}
          onDateChange={date => this.setState({date})}
        />
      </View>
    );
  }
}

export default DatePickerTest;

```

**Smartphone (please complete the following information):**
 - OS: ios
 - React Native version [0.61.0]
 - react-native-date-picker version [2.7.0]

**Comment by ptboyer** [comment]
Created a PR for this issue. :)

**Comment by henninghall** [comment]
Released in 2.7.1 (recommend you to install latest version 2.7.3)
